### PR TITLE
Modernize spec

### PR DESCRIPTION
**Pull request by dcermak**
Collection of small fixes:
- remove obsolete spec file entries like the group tag
- switch to more modern macros like `%make_build` and `%autosetup`

### CODE REVIEW

**Comment by perlpunk** [comment]
I just merged #24, so now there is a conflict. Could you rebase to current master?

**Comment by dcermak** [comment]
Tina Müller (tinita) <notifications@github.com> writes:

> I just merged #24, so now there is a conflict. Could you rebase to current master?

Done

**Comment by dcermak** [comment]
@perlpunk humble reminder :)

**Comment by perlpunk** [comment]
Sorry for the late reply. After checking on some details via Chat I activated the PR on the cronjob host.

File: cpanspec [hunk 1]
```diff
@@ -1474,19 +1474,42 @@ END
 $description
 
 \%prep
-\%setup -q@{[($noprefix ? "" : " -n $buildpath")]}
+END
+
+    my $all_patch_args_same = 1;
+    my $common_patch_args = undef;
```
Comment (perlpunk) [inline cpanspec:1480]: I have to change this in #55

**Pull request merged**